services/api/src/resources/organization/resolvers.ts

```diff
@@ -871,6 +872,64 @@ export const addGroupToOrganization: ResolverFn = async (
   return updatedGroup
 };
 
+// removes a user from all groups in an organisation
+export const removeUserFromOrganizationGroups: ResolverFn = async (
+  _root,
+  { input: { user: userInput, organization: organizationInput } },
+  { models, sqlClientPool, hasPermission, userActivityLogger }
+) => {
+
+  if (R.isEmpty(userInput)) {
+    throw new Error('You must provide a user id or email');
+  }
+
+  const user = await models.UserModel.loadUserByIdOrUsername({
+    id: R.prop('id', userInput),
+    username: R.prop('email', userInput)
+  });
+
+  // check the organization exists
+  const organizationData = await Helpers(sqlClientPool).getOrganizationById(organizationInput);
+  if (organizationData === undefined) {
+    throw new Error(`Organization does not exist`)
+  }
+
+  // check permissions and get groups
+  await hasPermission('organization', 'removeGroup', {
+    organization: organizationInput,
+  });
+  const orgGroups = await models.GroupModel.loadGroupsByOrganizationId(organizationInput);
```

Comment:
And change this to use this helper `models.GroupModel.loadGroupsByOrganizationIdFromGroups`, there is an example of how to use it here https://github.com/uselagoon/lagoon/blob/main/services/api/src/resources/organization/resolvers.ts#L325-L331